collector/pg_stat_statements.go

```diff
@@ -319,3 +361,28 @@ func (c PGStatStatementsCollector) Update(ctx context.Context, instance *instanc
 	}
 	return nil
 }
+
```

Comment:
Most of these 2 funcs are the same. Maybe we should consolidate to 1 function that just does the escaping and returns the slices.